site/content/404.md

```diff
@@ -1,13 +0,0 @@
----
-title: "404 - File not found"
-layout: 404
-description: ""
-url: /404.html
-robots: noindex,follow
```

Comment:
These 2 are the reasons we used the custom file. There's no other way around it, unfortunately...

Reply:
could add a block, near to `header` partial, But I suppose it's the same kind of trick so, if we need the noindex, lets close it

Reply:
Sounds good, sorry I didn't remember, but it's been so long.